bundles/org.openhab.binding.shelly/README.md

```diff
@@ -402,12 +402,14 @@ In this case the is no real measurement based on power consumption, but the Shel
 |          |returnedKWH  |Number   |yes      |Total returned energy, kwh                                                       |
 |          |reactiveWatts|Number   |yes      |Instantaneous reactive power, Watts                                              |
 |          |voltage      |Number   |yes      |RMS voltage, Volts                                                               |
+|          |powerFactor  |Number   |yes      |Power Factor                                                                     |
```

Comment:
Better use UoM here to make clear this is degree or percent.

Reply:
@fwolter this is a decimal value,   no degree or percent

Reply:
It's the ratio of active to apparent power. If I see correctly its range is 0-1 here, i.e. percent. It can also be expressed in degree, i.e. 0-90°. So, you should somehow make clear which notation you have chosen.

Reply:
you are right, I check the spec again, and it says "dimensionless"
Does the change from "Number" to "Number:Dimensionless" affects backward compatibility?

Reply:
I'm not 100% sure. As it's an informal parameter, which I guess is mostly visualized but not evaluated, I'd change it.

Reply:
I added the change